eu_dgc_v1_schema.json

```diff
@@ -97,22 +97,10 @@
                 },
                 "dob": {
                     "title": "Date of birth",
-                    "description": "Mandatory if no Person identifier is provided",
+                    "description": "The date of birth for of the person addressed in the certificate",
```

Comment:
minor: "The date of birth for of the person" -> "The date of birth of the person"

Reply:
Fixed in 8d5c2bacc6ce3e97b4f82c3377cd9b5985623f48